db/seeds.rb

```diff
@@ -1,109 +1,124 @@
 USERS_TO_CREATE = 20
+NUMBER = 0
 USERS_TO_CREATE.times do
-
-	User.create company:					Faker::Company.name,
-							email:						Faker::Internet.email,
-							password:	        Faker::Internet.password,
-							employees:				rand(8),
-							website:					Faker::Internet.url
-
+	NUMBER+=1
+	User.create({	company:					Faker::Company.name + "#{NUMBER}",
+								email:						"#{NUMBER}" + Faker::Internet.email,
+								password:	        Faker::Internet.password,
+								employees:				rand(8),
+								website:					Faker::Internet.url + "#{NUMBER}"
+							})
 end
 # works
 
-User.create company:					"Admin Man",
-						email:						"admin@admin.com",
-						password:	        "123456",
-						employees:				rand(8),
-						website:					Faker::Internet.url,
-						admin:						true
+User.create({ 	company:					"Admin Man",
+								email:						"admin@admin.com",
+								password:	        "123456",
+								employees:				rand(8),
+								website:					Faker::Internet.url,
+								admin:						true
+							})
 
-User.create company:					"Dev Man",
-						email:						"dev@dev.com",
-						password:	        "123456",
-						employees:				rand(8),
-						website:					Faker::Internet.url,
-						admin:						false
+User.create ({	company:					"Dev Man",
+								email:						"dev@dev.com",
+								password:	        "123456",
+								employees:				rand(8),
+								website:					Faker::Internet.url,
+								admin:						false
+							})
 
 business = ["Running", "Closed", "Updating"]
 5.times do
-    Location.create name:           Faker::Company.name,
-                  	address:        Faker::Address.street_address,
-                    city:           Faker::Address.city,
-                    postal_code:    Faker::Address.zip_code,
+	NUMBER+=1
```

Comment:
So Number =21 when it start off here right? Is that intended? Since it number starts at 0 and creates 20 Users first?

Confirmed with Brandon that this is intended for the entire seed